tests/cypress/integration/actions_tasks_objects/case_55_repeat_draw_feature.js

```diff
@@ -0,0 +1,162 @@
+        it('Draw and repeat draw a rectangle.', () => {
+            cy.createRectangle(createRectangleShape2Points);
+            cy.get('.cvat-canvas-container').trigger('mousemove', 200, 400);
+            cy.get('#cvat_canvas_shape_1').should('have.class', 'cvat_canvas_shape_activated');
+            cy.get('body').trigger('keydown', { keyCode: keyCodeN });
+            cy.get('.cvat-canvas-container')
+                .click(createRectangleShape2Points.firstX, createRectangleShape2Points.firstY - 200)
+                .click(createRectangleShape2Points.secondX, createRectangleShape2Points.secondY - 200);
+            cy.get('#cvat_canvas_shape_2').should('exist');
+            cy.get('.cvat_canvas_shape').then(($shape) => {
+                expect($shape.length).to.be.equal(2);
+            });
+            cy.get('.cvat-objects-sidebar-state-item').then(($sidebarItem) => {
+                expect($sidebarItem.length).to.be.equal(2);
+            });
+        });
+
+        it('Draw and repeat draw a polygon.', () => {
+            cy.createPolygon(createPolygonShape);
+            cy.get('.cvat-canvas-container').trigger('mousemove', 520, 400);
+            cy.get('#cvat_canvas_shape_3').should('have.class', 'cvat_canvas_shape_activated');
+            cy.get('body').trigger('keydown', { keyCode: keyCodeN });
+            createPolygonShape.pointsMap.forEach((element) => {
+                cy.get('.cvat-canvas-container').click(element.x, element.y - 200);
+            });
+            cy.get('.cvat-canvas-container').trigger('keydown', { keyCode: keyCodeN }).trigger('keyup');
+            cy.get('#cvat_canvas_shape_4').should('exist');
+            cy.get('.cvat_canvas_shape').then(($shape) => {
+                expect($shape.length).to.be.equal(4);
+            });
+            cy.get('.cvat-objects-sidebar-state-item').then(($sidebarItem) => {
+                expect($sidebarItem.length).to.be.equal(4);
+            });
+        });
+
+        it('Draw and repeat draw a polyline.', () => {
+            cy.createPolyline(createPolylinesShape);
+            cy.get('.cvat-canvas-container').trigger('mousemove', 700, 400);
+            cy.get('#cvat_canvas_shape_5').should('have.class', 'cvat_canvas_shape_activated');
+            cy.get('body').trigger('keydown', { keyCode: keyCodeN });
+            createPolylinesShape.pointsMap.forEach((element) => {
+                cy.get('.cvat-canvas-container').click(element.x, element.y - 200);
+            });
+            cy.get('.cvat-canvas-container').trigger('keydown', { keyCode: keyCodeN }).trigger('keyup');
+            cy.get('#cvat_canvas_shape_6').should('exist');
+            cy.get('.cvat_canvas_shape').then(($shape) => {
+                expect($shape.length).to.be.equal(6);
+            });
+            cy.get('.cvat-objects-sidebar-state-item').then(($sidebarItem) => {
+                expect($sidebarItem.length).to.be.equal(6);
+            });
```

Comment:
One of these checks is extra IMHO

Reply:
Removed it. Left the object sidebar check